### ISSUE TITLE
[BUG] Two word contractions aren't applied properly to capitalized definite articles in French

### ISSUE THREAD

**Issue by bdbernardy** [RosaeNLG/rosaenlg#196]
**Describe the bug**
Some French city names have a definite article: "Le Creusot", "Le Mans", "Les Mureaux", etc. Contractions aren't applied properly to to these proper nouns because of the capitalized "Le" or "Les".

"Je vais à Le Creusot" should be contracted to "Je vais au Creusot" for example.

**Expected behavior**
The following two word combinations should be contracted as follows:
-  'de' + 'Le' should be contracted to 'du'
- 'de' + 'Les' should be contracted to 'des'
- 'à' + 'Le' should be contracted to 'au'
- 'à' + 'Les' should be contracted to 'aux'

**Code**
The problem is in these [lines of code](https://github.com/RosaeNLG/rosaenlg/blob/47b39d3543e236cbc12014a141a7b3a39e12308e/packages/rosaenlg-filter/src/LanguageFilterFrench.ts#L40).

We need to add the following 4 elements to the two-word contraction rules:
```javascript
['de', 'Le', 'du'],
['de', 'Les', 'des'],
['à', 'Le', 'au'],
['à', 'Les', 'aux'],
```

**Context:**
 - RosaeNLG version: 4.2.0
 - Language: fr_FR
 - Packaging: node.js, browser

**Comment by ludans** [comment]
Thank @bdbernardy for spotting the issue and the lines to change. 
There is a small potential issue like in:

Ce principe permettra à Le Corbusier...

Which will be transformed as well. But let's consider that names should be properly protected with the adequate mixin. 

I'll probably do what you propose.

**Comment by ludans** [comment]
Thanks for the PR. Will be released in > 4.2.0.
Tell me if you need an early release (otherwise I will release in 1 or 2 months among other changes).

**Comment by bdbernardy** [comment]
> Thanks for the PR. Will be released in > 4.2.0. Tell me if you need an early release (otherwise I will release in 1 or 2 months among other changes).

Hi Ludans,

I was happy to help! 1-2 months is fine. Thank you for asking.
